Cargo.toml

```diff
@@ -30,9 +31,12 @@ dioxus = { version = "0.3", optional = true }
 cfg-if = "1.0.0"
 futures-util = {version = "0.3.28", optional = true }
 
+uuid = { version = "1.3.2", features = ["v4"], optional = true }
+async-broadcast = { version = "0.5.1", optional = true }
 clipboard = { version = "0.5.0", optional = true }
 notify-rust = { version = "4.8.0", optional = true }
 futures = { version = "0.3.28", features = ["std"], optional = true }
+log = "0.4.6"
```

Comment:
I don't think we need log, do we?

Reply:
Yeah I agree, I just changed things a bit, it's not really needed :))